ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/forms/useReportFormValues.tsx

```diff
@@ -60,13 +60,18 @@ export type ReportScope = {
 export type DeliveryDestination = {
     notifier: ReportNotifier | null;
     mailingLists: string[];
+    customSubject: string;
+    customBody: string;
 };
 
 export type ReportNotifier = {
     id: string;
     name: string;
 };
 
+export const maxEmailSubjectLength = 256;
+export const maxEmailBodyLength = 1000;
```

Comment:
I used these values because they were recommended by @zhenpesky from the mocks. @zhenpesky @charmik-redhat is there a better number you had in mind for the max character length for the email subject/body?

Reply:
> I wasn't part of any conversations where this was discussed, but is there any harm in letting the body be a bit longer? 1000 characters is only 4x the subject size, roughly two small paragraphs. Unless there is an API restriction or concern over database usage it seems like it would be beneficial to bump this up a little.

This part of the "body" is only the static text above the list of dynamic parameters for the report. It will be read once, the first time someone gets the report email, and then will be ignored for the rest of the report's existence. We can increase the length, but every additional line just pushes the important part of the email further down. I think this is a prime example of where "brevity is the soul of wit" (_Hamlet_, Act 2, Scene 2)

Reply:
True true. We talked to @zhenpesky and decided to increase it to 1500. I don't really know what the ideal number of characters is, but we can just add a few more just in case